### ISSUE TITLE
No diagHandler started in simulation mode

### ISSUE THREAD

**Issue by SGeeversAtVortech** [Deltares/rtc-tools#1062]
In GitLab by @MaartenS on Sep 3, 2018, 16:07

when using the PI-mixin in simulation problems, the diag.xml is not written by default. Basically, the following block of the optimization class in util.py is NOT available in simulation:

```python
if (issubclass(optimization_problem_class, PIMixin) and
            not any((isinstance(h, pi.DiagHandler) for h in logger.handlers))):
        handler = pi.DiagHandler(output_folder)
        logger.addHandler(handler)
```

Was there a good reason for this, and if not: can it be added?

**Comment by SGeeversAtVortech** [comment]
In GitLab by @vreeken on Dec 16, 2018, 20:20

unassigned @jvande42b

**Comment by SGeeversAtVortech** [comment]
In GitLab by @vreeken on Jan 14, 2019, 12:54

mentioned in commit 7ecac448b3c0aad133092d76fa121bde12c34e63

**Comment by SGeeversAtVortech** [comment]
In GitLab by @vreeken on Jan 31, 2019, 14:13

mentioned in commit ab4f7305974b4996f148d3086482bfaea7194468

**Comment by SGeeversAtVortech** [comment]
In GitLab by @vreeken on Jan 31, 2019, 15:30

mentioned in commit dbb4a4f361862441c91f16ca9b926cbeaae40d10

**Comment by SGeeversAtVortech** [comment]
In GitLab by @vreeken on Jan 31, 2019, 15:47

closed via merge request !247

**Comment by SGeeversAtVortech** [comment]
In GitLab by @vreeken on Jan 31, 2019, 15:47

closed via commit dbb4a4f361862441c91f16ca9b926cbeaae40d10
